var/spack/repos/builtin/packages/py-matplotlib/package.py

```diff
@@ -34,6 +34,8 @@ class PyMatplotlib(Package):
     homepage = "https://pypi.python.org/pypi/matplotlib"
     url      = "https://pypi.python.org/packages/source/m/matplotlib/matplotlib-1.4.2.tar.gz"
 
+    version('1.5.3', 'ba993b06113040fee6628d74b80af0fd',
+            url="https://pypi.python.org/packages/75/4e/2374eed18ac34421ccd7b4907080abd3009e112ca2c11b100c18961312e0/matplotlib-1.5.3.tar.gz")
```

Comment:
I think if we change the package's url to pypi.io instead of pypi.python.org (i.e., https://pypi.io/packages/source/m/matplotlib/matplotlib-1.4.2.tar.gz), then we won't have to specify the hashed url for every new version. In #1939 it sounds like some of the main packages support the pypi.io url.